### ISSUE TITLE
Move message prompt for waiting for pending job to DEBUG

### ISSUE THREAD

**Issue by gaow** [vatlab/SoS#893]
We had improved message for #871 on pending jobs:

```
Waiting for 1 pending job: output... ac3f2df01a7fb075.exe_info_. You can manually remove this lock file if you are certain that no other process is working on the output.
```

It is a long message with many file outputs. In the context of remote task it is a bit annoying because lost of jobs indeed take long time to queue or run. Shall we make this to `DEBUG` instead?

**Comment by gaow** [comment]
Well, then to make it less wordy maybe only report the name of the file in question that corresponds to the `exe_info`? It is just that for each step I have a lot of `_output` and they just keep showing up in groups on the screen ...

**Comment by BoPeng** [comment]
OK, you can shorten it as long as we know which lock file to check. The simplest one could be something like

```
Output xxx pending on lock yyy
```

**Comment by gaow** [comment]
The point is about narrowing down to the specific `_output[i]` responsible for the lock, when `concurrent=True`. Currently it shows the entire `_output`, which can be hundreds of files -- is my understanding correct?

**Comment by BoPeng** [comment]
[The message](https://github.com/vatlab/SoS/blob/master/src/sos/dag.py#L203) does not display output filename, only lock file name. You could leave only basename from lock filename and remove the 'manual removal' part altogether.

**Comment by BoPeng** [comment]
The lock is for each input group (`_output`) so there will be multiple messages for multiple input groups, not for multiple files in each input group.

**Comment by gaow** [comment]
Okey, but this variable `{pending_jobs[0]._signature[0]}` effectively displays lots of file names like `[file_target(a), file_target(b)]`. Now that I understand this is for each input group ... maybe we should limit its length using the same code you have for displaying input / output files in `INFO` at each step?

**Comment by BoPeng** [comment]
I see. You can use short_repr around it.

**Comment by gaow** [comment]
Getting back to it now ... Are you suggesting simply `{short_repr(pending_jobs[0]._signature[0])}`? I'll push a patch if this is exactly the right code?

**Comment by BoPeng** [comment]
Yes, but short repr might not be ready to shorten sos targets. You can try.

**Comment by gaow** [comment]
I did add `short_repr` but I do not think it has worked ...

**Comment by BoPeng** [comment]
I will need to create a test case do you have one already?

**Comment by gaow** [comment]
Oh wait, I'd have to update SoS on the remote host for this to happen, right? I can try that. But currently I'm having issues with remote host max job that I'll post in another ticket.

**Comment by BoPeng** [comment]
Remote `max_running_jobs` might have been fixed with yesterday's [batch size patch](https://github.com/vatlab/SoS/commit/e53611ec3a37b8cfeb042e2f7326b72d724b914c).

**Comment by gaow** [comment]
Oh okey let me upgrad e the host and test
